shared/src/Microsoft.Azure.Devices.Shared.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netstandard2.1;netstandard2.0;net472;net451</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netstandard2.1;netstandard2.0;net472;net451;netcoreapp2.1</TargetFrameworks>
```

Comment:
Need to mimic the same target frameworks as the service client library, so added netcoreapp2.1 here as well